Header header
uint8 note
uint8 velocity